CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2020-12-10
+### Changed
+- Allow usage of wildcards in subnet and security group names
```

Comment:
Could you please update it?

```suggestion
- Allow usage of wildcards in subnet and security group names. Thank you @RLRabinowitz (#41)
```
For markdown (#41) it should be `([#41](https://github.com/amplify-education/serverless-vpc-discovery/pull/41))`